daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v2/Runner.scala

```diff
@@ -54,7 +56,7 @@ private[lf] class Runner(
 
   def remapQ[X](result: Result[X, Free.Question, SExpr]): Result[X, ScriptF.Cmd, SExpr] =
     result.remapQ { case Free.Question(name, version, payload, stackTrace) =>
-      ScriptF.parse(name, version, payload, stackTrace) match {
+      ScriptF.parse(name, version, payload, knownPackages) match {
```

Comment:
We now pass up locations in multi-submit, so this context is needed in ScriptF parsing